metofficedatahub/multiple_files.py

```diff
@@ -99,3 +108,49 @@ def load_all_files(self) -> xr.Dataset:
         dataset = xr.merge(all_dataset)
 
         return dataset
+
+
+def save_to_zarr(dataset: xr.Dataset, save_dir: str, save_latest: bool = True):
+    """
+    Save dataset to zarr file
+
+    :param dataset: The Xarray Dataset to be save
+    :param save_dir: the directory where data is saved.
+        The zarr file will be saved using the timestamp of the run in isoformat
+    :param save_latest: option to save as 'latest.zarr'
+    """
+
+    logger.info(f"Saving data to zarr file here: {save_dir}")
+
+    # Make two files names
+    # 1. use the date timestamp of the data. Idea is that this will keep the historic
+    # 2. the latest - shows the most recent data without search through historic.
+    filename = pd.to_datetime(dataset.time.values).tz_localize("UTC").isoformat()
+    filename_and_path = f"{save_dir}/{filename}.zarr"
+    filename_and_path_latest = f"{save_dir}/latest.zarr/"
+
+    # extra step needed if we are saving to AWS.
+    # There may be different steps if saving to different file systems
+    if fsspec.open(save_dir).fs == s3fs.S3FileSystem():
+        filename_and_path = fsspec.get_mapper(
+            filename_and_path, client_kwargs={"region_name": "eu-central-1"}
+        )
+        filename_and_path_latest = fsspec.get_mapper(
+            filename_and_path_latest, client_kwargs={"region_name": "eu-central-1"}
+        )
+
+    # encoding
```

Comment:
```suggestion
```

I think it's fine to remove this comment as it's basically just repeating the variable name below

Reply:
I expanded a bit on the comment to keep it in